src/main/index.ts

```
@@ -187,3 +200,28 @@ ipcMain.handle('get-series-charCount-list', (_, date: { year: number; month: num
+      type: isDirectory ? 'folder' : 'file',
+      children: isDirectory ? readDirectoryRecursive(fullPath) : undefined
+    }
+  })
+}
+
+ipcMain.handle('read-series-structure', (_, seriesPath: string) => {
+  const rootPath = join(seriesPath, 'root')
+  try {
+    const structure = readDirectoryRecursive(rootPath)
+    return { success: true, structure }
+  } catch (err) {
```

ESLint

[error] 'err' is defined but never used.

(@typescript-eslint/no-unused-vars)